content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,78 @@
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** trust domain validation for services using mutual TLS. By default, the server only authenticates the requests from the same trust domain.
+- **Added** [labels](/docs/concepts/security/#how-citadel-determines-whether-to-create-service-account-secrets) to control service account secret generation by namespace.
+- **Added** SDS support to deliver the private key and certificates to each Istio control plane service.
+- **Added** support for [introspection](/docs/ops/troubleshooting/controlz/) to Citadel.
+- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
+- **Added** diagnostics to the Citadel Agent using the `/debug/sds/workload` and `/debug/sds/gateway` on port 8080.
+- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
+- **Improved** SDS security by enforcing the usage of [Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
+- **Improved** Citadel Agent logs by unifying the logging pattern.
+- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration in a future release.
+- **Enabled** the Envoy JWT filter by default to improv
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
+- **Added** alpha support for the OpenCensus trace driver.
+- **Improved** reporting for external services by removing requirements to add a service entry.
+- **Improved** the mesh dashboard to provide monitoring of Istio's configuration state.
+- **Improved** the Pilot dashboard to expose additional key metrics to more clearly identify errors.
+- **Removed** Kiali for 1.3 only. Kiali will be reintroduced to Itio as soon as [issue](https://github.com/istio/istio/issues/16972) is resolved.
```

Comment:
I don't think we should say its removed. Its still there, it may just not work properly.

Reply:
@rcaballeromx what type of keyword would you like for BDWNF.

Reply:
as Kiali was sorted out, I removed line 48.